packages/vscode-extension/src/webview/components/DeviceSelect.tsx

```diff
@@ -197,6 +205,14 @@ function DeviceSelect() {
     }
   };
 
+  const isAdminDisabledDeviceSections = ([label]: [
+    string,
+    IOSDeviceInfo[] | AndroidEmulatorInfo[] | AndroidPhysicalDeviceInfo[],
+  ]): boolean => {
+    const isRemoteAndroidAdminDisabled = useIsFeatureAdminDisabled(Feature.AndroidPhysicalDevice);
+    return label !== "Connected Android Devices" || !isRemoteAndroidAdminDisabled;
```

Comment:
I'd deMorgan the shit out of this

```suggestion
    return !(label === "Connected Android Devices" && isRemoteAndroidAdminDisabled);
```